FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Qemu_mps2/main_networking.c

```diff
@@ -145,12 +157,30 @@ void main_tcp_echo_client_tasks( void )
      * vApplicationIPNetworkEventHook() below).  The address values passed in here
      * are used if ipconfigUSE_DHCP is set to 0, or if ipconfigUSE_DHCP is set to 1
      * but a DHCP server cannot be contacted. */
-    FreeRTOS_debug_printf( ( "FreeRTOS_IPInit\n" ) );
```

Comment:
Why are we removing the prints here.

Reply:
Debug prints are not removed, they are added before the macro FREERTOS_PLUS_TCP_VERSION  is called in the Initialisation flow.